lib/straight/transaction.rb

```diff
@@ -0,0 +1,14 @@
+module Straight
+  Transaction = Struct.new(:tid, :amount, :confirmations, :block_height) do
```

Comment:
Because I don't need to inherit from an anonymous class. Better just make it named.
```
>> A = Struct.new(:a) #=> A
>> A.ancestors
=> [A, Struct, Enumerable, Object, …
>> class B < Struct.new(:b); end #=> nil
>> B.ancestors
=> [B, #<Class:0x0000000263e370>, Struct, Enumerable, Object, …
```